<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="1" total="4" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="用户管理系统测试套件" duration-ms="3082" started-at="2020-04-19T14:03:11Z" finished-at="2020-04-19T14:03:14Z">
    <groups>
      <group name="loginTrue">
        <method signature="LoginTest.loginTrue()[pri:0, instance:com.course.cases.LoginTest@4461c7e3]" name="loginTrue" class="com.course.cases.LoginTest"/>
      </group> <!-- loginTrue -->
    </groups>
    <test name="用户管理系统测试用例" duration-ms="3082" started-at="2020-04-19T14:03:11Z" finished-at="2020-04-19T14:03:14Z">
      <class name="com.course.cases.GetUserInfoTest">
        <test-method status="FAIL" signature="getUserInfo()[pri:0, instance:com.course.cases.GetUserInfoTest@77e4c80f]" name="getUserInfo" duration-ms="2028" depends-on-groups="loginTrue" started-at="2020-04-19T22:03:12Z" description="获取id为3的用户信息" finished-at="2020-04-19T22:03:14Z">
          <exception class="org.apache.ibatis.exceptions.PersistenceException">
            <message>
              <![CDATA[
### Error querying database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for 33
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for 33]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for 33
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for 33
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:30)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:150)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:141)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectOne(DefaultSqlSession.java:77)
	at com.course.cases.GetUserInfoTest.getUserInfo(GetUserInfoTest.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
Caused by: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for 33
	at org.apache.ibatis.session.Configuration$StrictMap.get(Configuration.java:875)
	at org.apache.ibatis.session.Configuration.getMappedStatement(Configuration.java:708)
	at org.apache.ibatis.session.Configuration.getMappedStatement(Configuration.java:701)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:147)
	... 27 more
]]>
            </full-stacktrace>
          </exception> <!-- org.apache.ibatis.exceptions.PersistenceException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getUserInfo -->
      </class> <!-- com.course.cases.GetUserInfoTest -->
      <class name="com.course.cases.LoginTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.course.cases.LoginTest@4461c7e3]" name="beforeTest" is-config="true" duration-ms="36" started-at="2020-04-19T22:03:11Z" description="测试准备工作" finished-at="2020-04-19T22:03:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="loginTrue()[pri:0, instance:com.course.cases.LoginTest@4461c7e3]" name="loginTrue" duration-ms="980" started-at="2020-04-19T22:03:11Z" description="用户登录成功接口" finished-at="2020-04-19T22:03:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTrue -->
      </class> <!-- com.course.cases.LoginTest -->
      <class name="com.course.cases.GetUserInfoListTest">
        <test-method status="FAIL" signature="getUserListInfo()[pri:0, instance:com.course.cases.GetUserInfoListTest@7fe8ea47]" name="getUserListInfo" duration-ms="14" depends-on-groups="loginTrue" started-at="2020-04-19T22:03:14Z" description="获取性别为男的用户信息" finished-at="2020-04-19T22:03:14Z">
          <exception class="org.apache.ibatis.exceptions.PersistenceException">
            <message>
              <![CDATA[
### Error querying database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for getUserLIstCase
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for getUserLIstCase]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for getUserLIstCase
### Cause: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for getUserLIstCase
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:30)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:150)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:141)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectOne(DefaultSqlSession.java:77)
	at com.course.cases.GetUserInfoListTest.getUserListInfo(GetUserInfoListTest.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
Caused by: java.lang.IllegalArgumentException: Mapped Statements collection does not contain value for getUserLIstCase
	at org.apache.ibatis.session.Configuration$StrictMap.get(Configuration.java:875)
	at org.apache.ibatis.session.Configuration.getMappedStatement(Configuration.java:708)
	at org.apache.ibatis.session.Configuration.getMappedStatement(Configuration.java:701)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:147)
	... 27 more
]]>
            </full-stacktrace>
          </exception> <!-- org.apache.ibatis.exceptions.PersistenceException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getUserListInfo -->
      </class> <!-- com.course.cases.GetUserInfoListTest -->
    </test> <!-- 用户管理系统测试用例 -->
  </suite> <!-- 用户管理系统测试套件 -->
</testng-results>
